ddtrace/contrib/django/patch.py

```diff
@@ -484,7 +484,8 @@ def _patch(django):
     # DEV: this check will be replaced with import hooks in the future
     if "django.conf.urls.static" not in sys.modules:
         import django.conf.urls.static
-    trace_utils.wrap(django, "conf.urls.url", traced_urls_path(django))
+    if django.VERSION < (4, 0, 0):
```

Comment:
Can we revert these changes since they're not related to the actual disabling/pinning of tests?

Reply:
oh sure, I was originally thinking "why not", will help keep the user's app from crashing, but since we cannot verify that we won't cause a runtime issue (rather than startup like this one), probably better to have a startup app crash